<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaVisualizzaRecensioneAmministratore</class>
 <widget class="QWidget" name="VistaVisualizzaRecensioneAmministratore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recensione - CineMax</string>
  </property>
  <widget class="QLabel" name="Sfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 1,
        x2: 0, y2: 0,
        stop: 0 #320F00,
        stop: 1 #641E00
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodiceFiscale">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Codice Fiscale Cliente:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitolo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Recensione</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ClickableLabel" name="labelHomeButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>63</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/HomeButtonAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelData">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDataRecensione">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStelle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Stelle:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStelleRecensione">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelOra">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Ora:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOraRecensione">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickableLabel" name="labelIndietroButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>63</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/IndietroButtonAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelRecensioneIcon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/RecensioneIconAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTesto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Testo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTestoRecensione">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #EB7864;
    color: #962D00;
	border: 1px solid #8C463C;
    border-radius: 4px;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodiceFiscaleRecensione">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #962D00;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
